02_knight_and_princess/dana_stepina/main.cpp

```diff
@@ -0,0 +1,115 @@
+#include <fstream>
+#include <iostream>
+#include <vector>
+
+using std::cout;
+using std::endl;
+
+using VertexId = int;
+using EdgeId = int;
+
+struct Vertex {
+  VertexId id;
+  std::vector<int> edges_id;
+};
+struct Edge {
+  EdgeId id;
+  int vertex_start;
+  int vertex_end;
+};
+
+class Graph {
+ public:
+  int count_edges = 0;
+  int count_vertices = 0;

```

Comment:
Зачем тебе вручную считать, когда это уже есть готовое?
```suggestion
  int count_vertices() const { return graph_vertices_.size(); }
```